Jenkinsfile

```diff
@@ -1,4 +1,6 @@
-private boolean lastCommitIsBumpCommit() {
+@Library('utils') _
+
+private boolean isBumpCommit() {
```

Comment:
this could go to the library as well as it's going to be shared right

Reply:
It's there just for a first testing iteration after the merge to develop branch. I am going to remove it after a successful publish without a loop.